### PR TITLE
Feature/15 selecting react

### LINKED ISSUE #15: React vs Angular

### PR DESCRIPTION
**Pull request by 3ZsForInsomnia**
Fixes #15

### CODE REVIEW

File: recommendations/details/reason.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 [ReasonML](https://reasonml.github.io/) is an alternative syntax to [OCaml](http://www.ocaml.org/) - a language that features functional, imperative, and object-oriented programming with a compiler that employs static typing and type inference. It was founded by the creator of ReactJS, [Jordan Walke](https://github.com/jordwalke), with the backing of Facebook, Bloomberg, and other individual contributors within the open source community.
 
+## Why not?
```
Comment (rlmartin) [inline recommendations/details/reason.md]: @3ZsForInsomnia @noahschechter I tried to summarize what I heard about this decision, but feel free to edit this (and/or below at https://github.com/ModaOperandi/agora/pull/26/files#diff-13426a425395f8b7c51ac5528c849e3dR30)

Comment (noahschechter) [reply recommendations/details/reason.md]: I think that's fine

**Comment by rlmartin** [comment]
Notes from conversation:
1. Display is largely migrated (already) on a branch.
2. Tooling for ReactML was lacking; for example: the linter would sometimes break code, the compiler error was simply "invalid syntax".
3. React: will be beneficial to boost internal expertise.
4. Angular: no one internally is passionate enough to fight for it.
5. React choice helps for hiring.
6. Need to develop a deprecation plan for Angular - FE guild is working on this.
7. Stylist app is on Angular 1, which has an EOL on the calendar; new FE engineer is starting on stylist app, and potentially will simply rewrite it.
8. Need to call angular as on Hold in this PR.

Tangent: micro-frontends
1. Strong packaging around different components.
2. Forces a separation of FE and API.
3. Goal is to position ourselves to move away from Mojo for FE.

**Comment by rlmartin** [comment]
Consensus from the group, though not reflected in Github approvals here.

**Pull request merged**